apis/pipelines/v1alpha5/run_conversion.go

```diff
@@ -1,3 +1,41 @@
 package v1alpha5
 
-func (*Run) Hub() {}
+import (
+	"github.com/sky-uk/kfp-operator/apis/pipelines"
+	hub "github.com/sky-uk/kfp-operator/apis/pipelines/v1alpha6"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+func (src *Run) ConvertTo(dstRaw conversion.Hub) error {
+	dst := dstRaw.(*hub.Run)
+	dstApiVersion := dst.APIVersion
+
+	err := pipelines.TransformInto(src, &dst)
+	if err != nil {
+		return err
+	}
+
+	dst.Spec.Provider = getProviderAnnotation(src)
+	dst.TypeMeta.APIVersion = hub.GroupVersion.String()
```

Comment:
how come you set this twice?

Reply:
this looks like a mistake, after refactoring, will remove one of the assignments